### PR TITLE
add DubboMultiParameterResolveServiceImplTest #866

### LINKED ISSUE #866: Add test DubboParamResolveService in Soul-web
- [ ] DefaultDubboParamResolveService  @Vincent-Xu 
- [ ] DubboMultiParameterResolveServiceImpl @candyYu

### PR DESCRIPTION
**Pull request by candyYu**
// Describe your PR here; eg. Fixes #issueNo
Fixes #866 

Make sure that:

- [ ] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [ ] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-web/src/test/java/org/dromara/soul/web/DubboMultiParameterResolveServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+/**
+ * Test cases for WebSocketParamFilter.
+ *
+ * @author candyYu
+ */
+@RunWith(MockitoJUnitRunner.class)
+public final class DubboMultiParameterResolveServiceImplTest extends DubboMultiParameterResolveServiceImpl {
```
Comment (dengliming) [inline soul-web/src/test/java/org/dromara/soul/web/DubboMultiParameterResolveServiceImplTest.java:36]: Why do we extends `DubboMultiParameterResolveServiceImpl ` here?

Comment (candyYu) [reply soul-web/src/test/java/org/dromara/soul/web/DubboMultiParameterResolveServiceImplTest.java:36]: i don't get it ,shoud i create a DubboMultiParameterResolveServiceImpl

Comment (dengliming) [reply soul-web/src/test/java/org/dromara/soul/web/DubboMultiParameterResolveServiceImplTest.java]: yeah. I suggest you new/mock the instance.

Comment (candyYu) [reply soul-web/src/test/java/org/dromara/soul/web/DubboMultiParameterResolveServiceImplTest.java]: i had new the instance with mock, please recheck

**Comment by yu199195** [comment]
You can add many test case.   like that  org.dromara.soul.examples.dubbo.api.service.DubboMultiParamService

**Review by yu199195 — APPROVED** [review]

**Pull request merged**